### PR TITLE
Update/refresh development application pages

### PR DESCRIPTION
**Pull request by steveoh**
this is a fresh start from https://github.com/agrc/gis.utah.gov/pull/2213. I think I have all the relevant changes. @nathankota please double check and make sure I didn't miss anything.

### CODE REVIEW

**Review by stdavis — DISMISSED** [review]
It looks good to me after a quick glance!

File: developer/applications/index.html [hunk 1]
```diff
@@ -121,48 +154,32 @@ <h6 class="text-center">Web Application Resources</h6>
...
               loading="lazy" />
           </a>
           <figcaption class="application-figure__caption application-figure__caption--large">State Parcel Map
           </figcaption>
         </figure>
       </td>
-      <td>esrijs, <a href="https://api.mapserv.utah.gov">WebAPI</a>, <a
-          href="{% link discover/index.html %}">Discover</a>, <a href="https://github.com/agrc/kitchen-sink">UGRC
-          widgets</a></td>
+      <td>ArcGIS Maps SDK for JavaScript, ReactJS via ViteJS, <a href="https://api.mapserv.utah.gov">UGRC API</a>, <a
```
Comment (steveoh) [inline developer/applications/index.html:172]:
```suggestion
      <td>ArcGIS Maps SDK for JavaScript, ReactJS, <a href="https://api.mapserv.utah.gov">UGRC API</a>, <a
```

**Review by steveoh — COMMENTED** [review]
everything looks good to me. 1 suggestion to remove vite and 1 suggestion to add another item to the main page to make the application symmetrical again.

**Comment by nathankota** [comment]
> everything looks good to me. 1 suggestion to remove vite and 1 suggestion to add another item to the main page to make the application symmetrical again.

remove vite suggestions committed, and I will try again on #2277 once this is merged and the roadkill image is available

**Comment by steveoh** [comment]
Is the road kill image available in this branch? no need for a whole new pull request.

**Comment by nathankota** [comment]
yes, the image is in this branch. apologies for the unnecessary branch + pr.

**Comment by steveoh** [comment]
i cherry picked your other commit to this branch. does it look good to you?

File: index.html [hunk 2]
```diff
@@ -897,12 +897,22 @@ <h5 class="text-center">Environmental Map (DEQ)</h5>
       <p class="muted">Search for site-specific information relating to DEQ programs including regulated environmental
         sites, facilities, and designations.</p>
     </div>
+    <div class="grid__col grid__col--1-of-2 text-center">
```
Comment (steveoh) [inline index.html:900]: since there are 3 columns now, this needs to match the upper stuff and be 1-of-3 in all the spots.

**Review by steveoh — COMMENTED** [review]
fix the css and this is good to go

**Comment by nathankota** [comment]
> fix the css and this is good to go

I'm happy with it if you are 👍

**Pull request merged**